### DISCUSSION TITLE
Homework JS IV

### DISCUSSION THREAD

**Discussion by PedroValenciano** [atralice/Curso.Prep.Henry#27] [General]
Hola, estoy atascado en este ejercicio:
function verificarPassword(usuario, password) {
  // Comprueba si la "password" enviada coincide con la propiedad "password" del objeto "usuario"
  // Devuelve "true" si coinciden
  // De lo contrario, devuelve "false"
  // // Tu código:
}
No entiendo cómo hacer para que el programa me pida una contraseña y la compare con la ya guardada

**Comment by atralice** [comment]
Hola, Pedro!
El `usuario` y `password` son strings que va a recibir tu función. En los tests se les pasa una string de prueba para ver si funciona tu código.

O sea, la función será usada así:
```js
verificarPassword(toni, 'hola');
```
Lo que tiene que hacer tu función es controlar que el objeto `toni` tenga una propiedad password cuyo valor  coincida con el password enviado como segundo parámetro de la función.
